src/test/java/uk/gov/hmcts/reform/sscs/service/AirLookupServiceTest.java

```diff
@@ -99,6 +99,15 @@ public void checkVenueForPostCodeWithEsaOrUcBenefitType(String postcode, String
         assertEquals(expectedVenue, venue);
     }
 
+    @Test
+    @Parameters({
+            "DE4 1SS, Chesterfield, industrialInjuriesDisablement"
+    })
+    public void checkVenueForPostCodeWithIidbBenefitType(String postcode, String expectedVenue, String benefitTypeCode) {
```

Comment:
The above 4 tests are doing exactly the same thing, it is duplicated code and since this is the case the tests names are a little confusing. Maybe rename one of the tests and merge the input parameters together.

Reply:
Done